Content.Shared/Foldable/DeployFoldableSystem.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Content.Shared.Buckle;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Interaction;
+
+namespace Content.Shared.Foldable;
+
+public sealed class DeployFoldableSystem : EntitySystem
+{
+    [Dependency] private readonly SharedHandsSystem _hands = default!;
+    [Dependency] private readonly SharedTransformSystem _transform = default!;
+    [Dependency] private readonly FoldableSystem _foldable = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<DeployFoldableComponent, AfterInteractEvent>(OnAfterInteract);
+    }
+
+    private void OnAfterInteract(Entity<DeployFoldableComponent> ent, ref AfterInteractEvent args)
+    {
+        if (args.Handled || !args.CanReach)
+            return;
+
+        if (!TryComp<FoldableComponent>(ent, out var foldable))
+            return;
+
+        if (!_hands.TryDrop(args.User, args.Used))
+            return;
+
+        _transform.SetCoordinates(args.Used, args.ClickLocation);
```

Comment:
is there a reason you're setting the coordinates manually instead of just passing `args.ClickLocation` into the `targetDropLocation` parameter of `TryDrop`? Doing it the other way seems to handle some of the edge cases better.